mlflow/tracking/_model_registry/client.py

```diff
@@ -128,19 +131,42 @@ def get_latest_versions(self, name, stages=None):
         """
         return self.store.get_latest_versions(name, stages)
 
+    def set_registered_model_tag(self, name, key, value):
+        """
+        Set a tag for the registered model.
+
+        :param name: Registered model name.
+        :param key: Tag key to log.
+        :param value: Tag value log.
+        :return: None
+        """
+        self.store.set_registered_model_tag(name, RegisteredModelTag(key, value))
+
+    def delete_registered_model_tag(self, name, key):
+        """
+        Delete a tag associated with the registered model.
+
+        :param name: Registered model name.
+        :param key: Registered model tag key.
+        :return: None
+        """
+        self.store.delete_registered_model_tag(name, key)
+
     # Model Version Methods
 
-    def create_model_version(self, name, source, run_id):
+    def create_model_version(self, name, source, run_id, tags=None):
         """
         Create a new model version from given source or run ID.
 
         :param name: Name ID for containing registered model.
         :param source: Source path where the MLflow model is stored.
-        :param run_id: Run ID from MLflow tracking server that generated the model
+        :param run_id: Run ID from MLflow tracking server that generated the model.
+        :param tags: A list of :py:class:`mlflow.entities.model_registry.ModelVersionTag`
```

Comment:
For run tags, client-level APIs accept a dictionary of tags and converts them to a list of `RunTag` objects before passing them to the backend store (see https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/_tracking_service/client.py#L88). I think we should do the same thing here.